File: DevExpress VCL 23.2.6 Full Source/VCL/ExpressEditors Library/Demos/Delphi/FormattedLabelDemo/HyperlinkDialog.pas
unit HyperlinkDialog;

interface

uses
  Windows, Messages, SysUtils, Variants, Classes, Graphics,
  Controls, Forms, Dialogs, cxGraphics, cxControls, cxLookAndFeels, cxLookAndFeelPainters, cxContainer,
  cxEdit, Menus, StdCtrls, cxButtons, cxTextEdit, cxMaskEdit, cxSpinEdit, cxButtonEdit, cxLabel, dxForms,
  dxLayoutcxEditAdapters, dxLayoutControlAdapters, dxLayoutContainer, cxClasses, dxLayoutControl;

type
  TfmHyperlinkDialog = class(TdxForm)
    btnOk: TcxButton;
    btnCancel: TcxButton;
    edtTextToDisplay: TcxTextEdit;
    edtAddress: TcxButtonEdit;
    dxLayoutControl1Group_Root: TdxLayoutGroup;
    dxLayoutControl1: TdxLayoutControl;
    dxLayoutGroup1: TdxLayoutGroup;
    dxLayoutGroup2: TdxLayoutGroup;
    dxLayoutGroup3: TdxLayoutGroup;
    dxLayoutItem1: TdxLayoutItem;
    dxLayoutItem2: TdxLayoutItem;
    dxLayoutGroup4: TdxLayoutGroup;
    dxLayoutItem3: TdxLayoutItem;
    dxLayoutItem4: TdxLayoutItem;
  end;

implementation

{$R *.dfm}

end.

File: DevExpress VCL 23.2.6 Full Source/VCL/ExpressEditors Library/Demos/CBuilder/FormattedLabelDemo/HyperlinkDialog.cpp
//---------------------------------------------------------------------------

#include <vcl.h>
#pragma hdrstop

#include "HyperlinkDialog.h"
//---------------------------------------------------------------------------
#pragma package(smart_init)
#pragma link "cxButtonEdit"
#pragma link "cxButtons"
#pragma link "cxClasses"
#pragma link "cxContainer"
#pragma link "cxControls"
#pragma link "cxEdit"
#pragma link "cxGraphics"
#pragma link "cxLookAndFeelPainters"
#pragma link "cxLookAndFeels"
#pragma link "cxMaskEdit"
#pragma link "cxTextEdit"
#pragma link "dxLayoutContainer"
#pragma link "dxLayoutControl"
#pragma link "dxLayoutControlAdapters"
#pragma link "dxLayoutcxEditAdapters"
#pragma resource "*.dfm"
TfmHyperlinkDialog *fmHyperlinkDialog;
//---------------------------------------------------------------------------
__fastcall TfmHyperlinkDialog::TfmHyperlinkDialog(TComponent* Owner)
	: TForm(Owner)
{
}
//---------------------------------------------------------------------------
